operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -113,6 +116,61 @@ protected Status createStatus() {
                 })));
     }
 
+    @Test
+    public void testSuccessfulReconcileWithSelectorLabel(VertxTestContext context)  {
+        MetricsProvider metrics = createCleanMetricsProvider();
+
+        AbstractWatchableStatusedResourceOperator resourceOperator = resourceOperatorWithExistingResource();
+
+        Labels nsLabel = Labels.fromMap(Collections.singletonMap("nsLabelKey", "nsLabelValue"));
+
+        AbstractOperator operator = new AbstractOperator(vertx, "TestResource", resourceOperator, metrics, nsLabel) {
+            @Override
+            protected Future createOrUpdate(Reconciliation reconciliation, CustomResource resource) {
+                return Future.succeededFuture();
+            }
+
+            @Override
+            public Set<Condition> validate(Reconciliation reconciliation, CustomResource resource) {
+                return emptySet();
+            }
+
+            @Override
+            protected Future<Boolean> delete(Reconciliation reconciliation) {
+                return null;
+            }
+
+            @Override
+            protected Status createStatus() {
+                return new Status() { };
+            }
+        };
+
+        assertThat(operator.getMatchLables(), is(nsLabel.toMap()));
+
+        Checkpoint async = context.checkpoint();
+        operator.reconcile(new Reconciliation("test", "TestResource", "my-namespace", "my-resource"))
+                .onComplete(context.succeeding(v -> context.verify(() -> {
+                    MeterRegistry registry = metrics.meterRegistry();
+
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.successful").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.failed").tag("kind", "TestResource").counter().count(), is(0.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.locked").tag("kind", "TestResource").counter().count(), is(0.0));
+
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.duration").tag("kind", "TestResource").timer().count(), is(1L));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.duration").tag("kind", "TestResource").timer().totalTime(TimeUnit.MILLISECONDS), greaterThan(0.0));
```

Comment:
You seem to test the counters. But I do not see the labels used anywhere here. Should it be asserted somewhere as well?